src/graphics/vulkan/vulkan_command_buffer.cpp

```
@@ -251,6 +312,25 @@ namespace canyon::graphics::vulkan {
         vkCmdCopyImage(m_vkCommandBuffer, srcImage.GetVkImage(), VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL, dstImage.GetVkImage(), VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL, 1, &region);
     }
 
+    void CommandBuffer::HostWriteToVertexBarrier(Buffer const& buffer) {
```

Clang (14.0.6)

[error] method 'HostWriteToVertexBarrier' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)